docs/recommendations/protein/delins.md

```diff
@@ -28,9 +28,9 @@ bin/pull-syntax -c -f docs/syntax.yaml aa.delins
     - **encode a translation stop codon** in the inserted sequence are described as deletion-insertion of this sequence, not as a deletion-insertion removing the entire C-terminal amino acid sequence.
     - encode an open reading frame which **after** the inserted sequence shifts to another reading frame, are described as a [frameshift](frameshift.md).
     - **start N-terminal (5') of and including** the translation termination (stop) codon, are described as [frameshift](frameshift.md).
-- fusion proteins represent a special case of deletion-insertion variant.
-  The fusion break point is described using **`::`**.<br>
-  **NOTE**: to avoid confusion, HGVS recommends to follow the [HGNC guidelines](https://www.genenames.org/about/guidelines/) to describe products of gene translocations or fusions (format `GENESYMBOL1::GENESYMBOL2`) and readthrough transcripts (format `GENESYMBOL1-GENESYMBOL2`).
```

Comment:
This same sentence is still on the RNA delins page. For consistency, shouldn't both sentences be adjusted? However, I'm wondering about the examples given (`GENESYMBOL1::GENESYMBOL2` and `GENESYMBOL1-GENESYMBOL2`). They are now removed; are they not relevant here? The adjoined transcripts page only lists the `GENE::GENE` example, not the `GENE-GENE` readthrough format. The RNA splicing page and the general page still list both examples. Perhaps we should clarify somewhere how adjoined transcripts are different from readthrough transcripts?